combiner/lists/TeX-MML-AM_CHTM-fullL.lis

```diff
@@ -0,0 +1,22 @@
+jax/input/TeX/config.js
```

Comment:
The name of this file has an extra `L` after `-full` (`TeX-MML-AM_CHTM-fullL.lis`).  The file needs to be renamed.